tools/webpack-loaders/svg-loader/index.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+
+module.exports = function inlineSvgLoader(content) {
+    // eslint-disable-next-line no-unused-expressions
+    this.cacheable && this.cacheable(); // webpack-specfic weirdness
```

Comment:
I was reading this morning that this is the default in webpack 2, so not needed here.